build/ci/unit-tests.yml

```diff
@@ -38,102 +38,23 @@ pr:
     - README.md
 
 jobs:
-- job: Windows
-  pool:
-    name: NetCorePublic-Pool
-    queue: buildpool.windows.10.amd64.vs2019.pre.open
-  strategy:
-    maxParallel: 4
-    matrix:
-      Debug:
-        _configuration: Debug
-      Release:
-        _configuration: Release
-  timeoutInMinutes: 20
-  steps:
-    - script: $(Build.SourcesDirectory)\build.cmd /build /test /ci /sign /diagnostic /no-deploy /no-integration /no-ibc /configuration $(_configuration)
-    - task: PublishBuildArtifacts@1
-      inputs:
-        PathtoPublish: '$(Build.SourcesDirectory)\artifacts\$(_configuration)\log'
-        ArtifactName: '$(_configuration) unit test logs'
-        publishLocation: Container
-      continueOnError: true
-      condition: failed()
-    - task: PublishBuildArtifacts@1
-      inputs:
-        PathtoPublish: '$(Build.SourcesDirectory)\artifacts\$(_configuration)\bin'
-        ArtifactName: '$(_configuration) bin folder'
-        publishLocation: Container
-      continueOnError: true
-      condition: failed()
-    - task: PublishBuildArtifacts@1
-      inputs:
-        PathtoPublish: '$(Build.SourcesDirectory)\artifacts\$(_configuration)\VSSetup'
-        ArtifactName: '$(_configuration) VSSetup folder'
-        publishLocation: Container
-      continueOnError: true
-      condition: failed()
-    - task: PublishBuildArtifacts@1
-      inputs:
-        PathtoPublish: '$(Build.SourcesDirectory)\artifacts\$(_configuration)\TestResults'
-        ArtifactName: '$(_configuration) Test Result Logs'
-        publishLocation: Container
-      continueOnError: true
-      condition: failed()
-    - task: PublishTestResults@2
-      inputs:
-        testRunner: 'xUnit'
-        testResultsFiles: '**/*.xml' 
-        searchFolder: '$(Build.SourcesDirectory)\artifacts\$(_configuration)\TestResults'
-        configuration: '$(_configuration)'
-        publishRunAttachments: true
-        failTaskOnFailedTests: true
-      continueOnError: true
-      condition: always()
+- template: unit-tests-template.yml
+  parameters:
+    jobName: Windows
+    poolName: NetCorePublic-Pool
+    poolQueue: BuildPool.Windows.10.Amd64.VS2019.Pre.Open
+    buildConfiguration: Debug
 
-- job: Spanish
-  pool:
-    name: NetCorePublic-Pool
-    queue: BuildPool.Windows.Amd64.VS2019.Pre.ES.Open
-  variables:
-    _configuration: Debug
-  timeoutInMinutes: 20
-  steps:
-    - script: $(Build.SourcesDirectory)\build.cmd /build /test /ci /diagnostic /no-sign /no-deploy /no-integration /no-ibc /clearnugetcache /configuration $(_configuration)
```

Comment:
This used to pass `/no-sign` but will now use `/sign`.

Reply:
See description.